synapseclient/client.py

```diff
@@ -1259,7 +1381,17 @@ def _download_file_entity(
         entity: Entity,
         ifcollision: str,
         submission: str,
-    ):
+    ) -> None:
+        """
+        Download file entity
+
+        Arguments:
+            downloadLocation: The download location
+            entity:           The Synapse Entity object
+            ifcollision:      Determines how to handle file collisions.
```

Comment:
It might look best to show the options in an unordered list. 

Here is an example: https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/synapseclient/client.py#L3418

Reply:
Great question. I don't have a strong preference on either direction personally.

Though if we are thinking about consistency all of the attributes are highlighted in grey. `includeEntityTypes` in your screenshot. 

It might make the most sense to keep all options highlighted in grey.

Reply:
Updated options in the unordered lists.